### ISSUE TITLE
Nothing but commons?

### ISSUE THREAD

**Issue by KSUWildcatFan** [AHAAAAAAA/PokemonGo-Map#2864]
I'm having results populate into my map, but it seems to be nothing but commons. Don't get me wrong, this area isn't exactly a hotbed for rare pokemon but usually there's at least ONE good Pokemon within a 15-step radius at any given time. Is it possible Niantic figured out how to filter somehow, and my account is blacklisted? Even my Magikarp spawn location around the corner doesn't seem to be popping up on my  map, but I've plucked a few off the in-game tracker today.

**Comment by KSUWildcatFan** [comment]
python runserver.py -a ptc -u UID -p PWD -st 15 -k gmapkey -l "39.015884, -95.588537" -sd 5

That's what I'm running.

**Comment by antiremy** [comment]
Are you running one thread? Whenever I ran more than 1 thread and 5 step delay, pokemon dont show up

**Comment by pokegomon** [comment]
Im getting errors in my terminal for a dictionary error and also see only common pokemon

**Comment by jesusdrodriguez** [comment]
add -t 1 -sd 5 to your command arguments.

**Comment by KSUWildcatFan** [comment]
Had the -sd 5 but not the -t 1. I was running -t 3 at one point. I had heard at one point that Niantic might be flagging accounts that seemed to be bots and that moving the search location 100 yards away could help..seemed to help yesterday (on the old version) so maybe I'll try that too.

**Comment by Crazy-Clown** [comment]
> Im getting errors in my terminal for a dictionary error and also see only common pokemon

You're scanning too fast.

**Comment by antiremy** [comment]
@KSUWildcatFan Niantic lowered the request rate to 1 per 5 seconds, hence the 1 thread 5 step delay

**Comment by pokegomon** [comment]
thank you! i will try to remedy that the -t 1 -sd 5 seems to work.

But I didnt have to do that yesterday

**Comment by pokegomon** [comment]
oh!!! thanks again!!!

**Comment by KSUWildcatFan** [comment]
Yeah, I figured the -t 3 was screwing things up earlier but haven't been running that for some time.

I moved my marker down the street and set -t 1 -sd 5. We'll see what pops up.

**Comment by Crazy-Clown** [comment]
I'm using the following in my config:

num-threads   1
scan-delay  2
step-limit   5  

Works without issues.

**Comment by KSUWildcatFan** [comment]
Tracking says there's a Scyther nearby (just restarted my app) but nothing on the map. Blah.

**Comment by antiremy** [comment]
Did you let the search complete? It takes forever with the new settings

**Comment by Crazy-Clown** [comment]
> I'm having results populate into my map, but it seems to be nothing but commons. Don't get me wrong, this area isn't exactly a hotbed for rare pokemon but usually there's at least ONE good Pokemon within a 15-step radius at any given time. Is it possible Niantic figured out how to filter somehow, and my account is blacklisted? Even my Magikarp spawn location around the corner doesn't seem to be popping up on my map, but I've plucked a few off the in-game tracker today.

It's quite possible that your 15 step radus is way too high.  There's a chance that your map hadnt updated by the time the scan originally started, and you walked right into the magicarp as the scanner was in the high 10-15 radius area when the pokemon was in the low end.

I like to run it no higher than 5, and have multiple that way it updates frequently and I make sure I have overlapping so the update stays current

**Comment by KSUWildcatFan** [comment]
I dropped it down to 5. Still junk. Not sure it's done yet though. I'll keep watching.

**Comment by Crazy-Clown** [comment]
> I dropped it down to 5. Still junk. Not sure it's done yet though. I'll keep watching.

After observing the output of my maps for the past week and a bit - honestly, I'm not surprised that there is so much crap there and very little uncommons.  Let's just hope that they don't have a way to detect the scanning and then only display common crap lol.

**Comment by KSUWildcatFan** [comment]
My area has been decent most days. Today, the rarest thing that's popped up has been a Pinsir. Otherwise, just the common fodder. Scyther never popped up and disappeared from Nearby when I reloaded my app.

**Comment by pstinsonnc** [comment]
I'm having a similar experience. I've already adjusted the scanning speed. I'm finding plenty of Pokemon, but it's all Pidgey/Ratata/Spearow/Weedle etc. I live in a small town, mostly rural, but there's usually at least one or two decent Pokemon within a 10 step radius. I'm wondering if it's just due to Niantic changing up geographical spawn locations.

**Comment by dmtdabs** [comment]
I am also experiencing similar issues where not as many Rare pokemon appear on my map. Don't get me wrong, Ill still get about 1-2 rares a day, but its not as frequent as it has been in the past. Most of the time i see commons on the map.

Ive been running the settings with:
Step Limit 8
Scan Delay 2
Threading 1

Im going to try running the Step Limit 5, Scan Delay 5 and Threading 1 to see what results i get

**Comment by KSUWildcatFan** [comment]
Nothing I've tried has made my results better. I did have a Beedrill and a Rhyhorn pop up but that's about it. Nothing terribly 'rare' per se. It sure does seem like Niantic has this figured out..for now.

**Comment by kipperjx2** [comment]
Same here, -st 5, -sd 5, -t 1. Nothing but Commons.....

**Comment by KSUWildcatFan** [comment]
I punched in coordinates for my work (downtown, business area) and it certainly does seem to be pulling up a lot "more" pokemon than my home area, and there are a few that aren't crap (Scyther, Magikarp)..I'm not sure why my home area isn't mapping worth a damn but it could also be that they just made spawns suck here, too...It's pretty barren now. That said, with the map not seeming to populate everything, I have no idea if it's not showing everything or there's just not much to show.

**Comment by femot** [comment]
Running this config.ini with 6 accounts (on `develop` branch)
```
num-threads: 6
scan-delay: 5.1
step-limit: 12
```

I got some Pokemon that are "Rare", but nothing really rare. That being said, my town is always kinda bad with spawns.

Those are my current stats:

Total	122	
	Bellsprout	1	0.8%
	Caterpie	6	4.9%
	Drowzee	1	0.8%
	Eevee	6	4.9%
	Fearow	1	0.8%
	Gastly	1	0.8%
	Golbat	1	0.8%
	Krabby	1	0.8%
	Magikarp	2	1.6%
	Nidoran♂	1	0.8%
	Paras	3	2.5%
	Pidgeotto	1	0.8%
	Pidgey	20	16.4%
	Poliwag	1	0.8%
	Raticate	1	0.8%
	Rattata	24	19.7%
	Spearow	6	4.9%
	Squirtle	1	0.8%
	Staryu	2	1.6%
	Venonat	4	3.3%
	Weedle	36	29.5%
	Zubat	2	1.6%

**Comment by juliusrucker25** [comment]
Ive seen two Marowak's on my map today (first time ever in my town for me) and both were 1 min spawn times. Is this the new spawn times for the less common pokemon? If so, that sucks big time.

**Comment by KSUWildcatFan** [comment]
I haven't seen much quality at all pop up around my area. Downtown looked more interesting with several Scyther and a Growlithe. If spawn times are shorter, with no real tracking available, then this is a complete joke of an app and it's spiraling out of control rapidly.

**Comment by jesusdrodriguez** [comment]
@KSUWildcatFan blame Niantic not the app.

**Comment by KSUWildcatFan** [comment]
Oh, I do. I was referring to Niantic, not this script. I wasn't very clear, sorry.

Niantic can suck a fat one.
